test-clients/src/main/java/com/hedera/services/bdd/suites/perf/PerfTestLoadSettings.java

```diff
@@ -26,11 +26,19 @@
 public class PerfTestLoadSettings {
 	public static final int DEFAULT_TPS = 500;
 	public static final int DEFAULT_TOLERANCE_PERCENTAGE = 5;
-	public static final int DEFAULT_MINS = 5;
+	public static final int DEFAULT_MINS = 1;
 	public static final int DEFAULT_ALLOWED_SECS_BELOW = 60;
 	public static final int DEFAULT_BURST_SIZE = 5;
 	public static final int DEFAULT_THREADS = 50;
 	public static final int DEFAULT_SUBMIT_MESSAGE_SIZE = 256;
+	public static final int DEFAULT_SUBMIT_MESSAGE_SIZE_VAR = 64;
+	// By default, it will fall back to original test scenarios
+	public static final int DEFAULT_TOTAL_TEST_ACCOUNTS = 2;
+	public static final int DEFAULT_TOTAL_TEST_TOPICS = 1;
+	public static final int DEFAULT_TOTAL_TEST_TOKENS = 1;
+	public static final int DEFAULT_TOTAL_TEST_TOKEN_ACCOUNTS = 2;
+	public static final int DEFAULT_TEST_TREASURE_START_ACCOUNT = 1001;
+	public static final int DEFAULT_TOTAL_CLIENTS = 1;
```

Comment:
Please provide comment for `totalClients`, why the property is use.

Reply:
Done.